packages/gatsby-source-medium/README.md

```diff
@@ -0,0 +1,56 @@
+# gatsby-source-medium
+
+Source plugin for pulling data into Gatsby from unofficial Medium JSON endpoints.
+
+## Install
+
+`npm install --save gatsby-source-medium`
```

Comment:
Not necessary once it is part of gatsby :)

Reply:
It still is, though ;) This is a [lerna](https://lernajs.io/) - so all packages live in the same repository, but are distributed as individual npm packages.